### PR TITLE
privacy.sexy: Add version 0.11.4

### PR DESCRIPTION
**Pull request by Zliced13**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/privacy.sexy.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.11.4",
+    "homepage": "https://privacy.sexy/",
+    "description": "An open-source tool to enforce privacy & security best-practices on Windows & macOS, because privacy is sexy 🍑🍆",
```
Comment (HUMORCE) [inline bucket/privacy.sexy.json:4]:
```suggestion
    "description": "Enforce privacy & security on Windows.",
```

**Comment by Zliced13** [comment]
@HUMORCE can you please also accept my Zoom PR. It's a pretty major one because a lot of people depend on Zoom for their daily lives.

**Pull request merged**